tests/Integration/Behaviour/Features/Scenario/Module/module.feature

```diff
@@ -50,6 +50,25 @@ Feature: Module
       | enabled        | true                 |
       | installed      | true                 |
 
+  Scenario: Uninstall modules
+    Given module ps_featuredproducts has following infos:
+      | technical_name | ps_featuredproducts |
+      | version        | 1.0.0               |
+      | enabled        | true                |
+      | installed      | true                |
+    Given module ps_emailsubscription has following infos:
+      | technical_name | ps_emailsubscription |
+      | version        | 1.0.0                |
+      | enabled        | true                 |
+      | installed      | true                 |
+    When I bulk uninstall modules: "ps_featuredproducts,ps_emailsubscription"
+    And module ps_featuredproducts has following infos:
+      | installed      | false               |
+    Then I should have an exception that module is not installed
```

Comment:
If you remove the exception then you won't need this step anymore Adn the `false` value above will be correctly compared with the returned value in the query result